rbac/migration_tool/migrate.py

```diff
@@ -119,9 +105,10 @@ def migrate_role(role: Role):
     v1_to_v2_mapping = shared_system_role_replicated_role_bindings_v1_to_v2_mapping
     permissioned_role_migrator = Migrator(v1_to_v2_mapping)
     v2_roles = [v2_role for v2_role in permissioned_role_migrator.migrate_v1_roles(v1_role)]
-    spicedb_rel_summary = spicedb_relationships(frozenset(v2_roles))
-    for rel in spicedb_rel_summary:
+    relationships = spicedb_relationships(frozenset(v2_roles))
+    for rel in relationships:
         logger.info(stringify_spicedb_relationship(rel))
+    # write_relationships(relationships)
```

Comment:
1. Maybe we could have environment variable to be enabled/disable - or maybe rather add parameter in request ? (or both :) )

2. If we will run it for all orgs all - it would lot of relations and I curious what would happen :) and I would definitely try it :) 

We can also implement some mechanisms like batching with retry logic.

3. Maybe it would be great to log how many relation are we going to write.

Reply:
Updated with a parameter, which seems more convinient